dags/overwatch.py

```diff
@@ -0,0 +1,53 @@
+"""
+Overwatch
+Runs daily at 0700 UTC
+
+Source code is [overwatch repository](https://github.com/mozilla/overwatch/).
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models import Variable
+from operators.gcp_container_operator import GKEPodOperator
+
+docs = """\
+This DAG is executes the Overwatch monitoring system.
```

Comment:
I'd move all of this to the module doc string above and just just the docs in the DAG object to `__doc__`

Reply:
https://stackoverflow.com/a/7791595